src/web/mjs/connectors/MangaNato.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class MangaNato extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'manga-nato';
+        super.label = 'MangaNato';
```

Comment:
May add a space to better distinguish between `Manganato` (aka MangaNel)

<img width="458" alt="image" src="https://user-images.githubusercontent.com/27136773/164884105-7f4a0a3c-bb5b-4dc5-b0fb-48e11772055c.png">

Reply:
Renamed to Manga Nato